### PR TITLE
Enable trailing slashes for docs/website

### PR DESCRIPTION
**Pull request by ntolia**
## Description

This should fix a bunch of the errors seen by Google when indexing the website.

## Type of change

- [x] :bug: Bugfix
- [x] :world_map: Documentation

### CODE REVIEW

**Comment by ntolia** [comment]
This is a lot more complicated than I had thought.

File: docs/docs/intro.md [hunk 1]
```diff
@@ -12,6 +12,6 @@ beyond M365, will expand based on the interest and needs of the community.
 
 ## Getting started
 
-You can follow the [Quick Start](quickstart) guide for an end-to-end Corso walk through. Alternatively, follow
-the instructions in the [Corso Setup](setup/concepts) section to dive into the details on how to configure and
+You can follow the [Quick Start](../quickstart) guide for an end-to-end Corso walk through. Alternatively, follow
+the instructions in the [Corso Setup](../setup/concepts) section to dive into the details on how to configure and
```
Comment (gmatev) [inline docs/docs/intro.md:16]: @ntolia should this have the trailing slash as above?

File: docs/blog/2022-11-16-corso-announcement-free-backup-for-microsoft-365.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ costs, as well.
 ## Interested in Trying Corso?
 
 Corso, currently in alpha, provides a CLI-based tool for backups of your M365 data.
-[Follow the quickstart guide](../docs/quickstart) to start protecting your business-critical M365 data in
+[Follow the quickstart guide](../../docs/quickstart/) to start protecting your business-critical M365 data in
```
Comment (gmatev) [inline docs/blog/2022-11-16-corso-announcement-free-backup-for-microsoft-365.md:79]: the trailing slash pattern here is different from what I see below. Likely one of the two is off.

Comment (ntolia) [reply docs/blog/2022-11-16-corso-announcement-free-backup-for-microsoft-365.md:79]: Ah, somewhat of an accidental change that is actually a no-op.

Comment (ntolia) [inline docs/blog/2022-11-16-corso-announcement-free-backup-for-microsoft-365.md:79]:
```suggestion
[Follow the quickstart guide](../../docs/quickstart) to start protecting your business-critical M365 data in
```

**Review by gmatev — APPROVED** [review]

File: docs/docs/intro.md (hunk 1 above)
Comment (ntolia) [reply docs/docs/intro.md:16]: Fixed the above issue but it's a no-op with this setting.

**Pull request merged**